Dockerfile

```diff
@@ -0,0 +1,21 @@
+#
+# BUILD CONTAINER
+#
+FROM node:16 as base
+USER node
```

Comment:
Do we need to set a user?

Reply:
I think it's better to run containers as non-root user (the default): https://docs.bitnami.com/tutorials/why-non-root-containers-are-important-for-security